articles/azure-monitor/app/sdk-connection-string.md

```diff
@@ -90,7 +90,12 @@ See also: [Regions that require endpoint modification](./custom-endpoints.md#reg
 - [Profiler](./profiler-overview.md): `profiler`
 - [Snapshot](./snapshot-debugger.md): `snapshot`
 
+#### Is Connection String a secret?
 
+Connection String contains iKey which is a unique identifier used by the ingestion service to associate telemetry to a specific Application Insights resource.  It is not to be considered a security token or key. The ingestion endpoint provides [AAD based authenticated telemetry ingestion options] if you want to protect your AI resource from misuse.
```

Comment:
Having this `[AAD based authenticated telemetry ingestion options]` I think is going to cause questions about supporting this for the web clients which doesn't support this...